src/main/resources/application.conf

```diff
@@ -0,0 +1,2 @@
+  json = "//file.json"
+  channels = "src/main/resources/channelsId.txt"
```

Comment:
it's ok to store ids in file however, if you suggest that this path parameter would be changed a lot, this path from main args will suite better.
And rename it:)